.github/labeler.yml

```
@@ -0,0 +1,2 @@
+needs-triage:
+  - '*'
```

```suggestion
needs-triage:
  - '.*'
```
`*` is not valid regex, as it attempts to quantify a token that doesn't exist. If issue-labeler somehow interprets '*' as a glob pattern then perhaps it's still functional and this could be ignored.

Otherwise, to match everything we should use either `.*` or an empty regex. Both seem to result in a match for any string in JS, which I believe is the flavor of regex that applies to GitHub Actions.

---

oops